docs/examples/composer_app/composer.json

```diff
@@ -0,0 +1,6 @@
+{
+    "require": {
+        "monolog/monolog": "1.3",
+        "marcelog/pami": "dev-master"
```

Comment:
Do you mind if I verify with a completely clean installation before merging? Just to make sure that it actually works out of the box.